Update documentation of `azure_rm_virtualnetwork` to reflect that the `dns_servers` limit on length is no longer 2.

kmj251 opened the pull request:
##### SUMMARY
<!--- Describe the change below, including rationale and design decisions -->
Updated the documentation for the `azure_rm_virtualnetwork` module to reflect that the dns_servers variable doesn't have a limit on the length of the list.

<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
Fixes #1081 

##### ISSUE TYPE
<!--- Pick one below and delete the rest -->
- Docs Pull Request

##### COMPONENT NAME
<!--- Write the short name of the module, plugin, task or feature below -->

##### ADDITIONAL INFORMATION
<!--- Include additional information to help people understand the change here -->
<!--- A step-by-step reproduction of the problem is helpful if there is no related issue -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```paste below
The documentation says that the dns_servers list variable has a maximum length of 2. There is no limit to the number of DNS servers, so the documentation is incorrect.
```

kmj251 committed:
Update documentation of `azure_rm_virtualnetwork` to reflect that the `dns_servers` limit on length is no longer 2.

kmj251 commented:
Resolves #1081

kmj251 committed:
Updating version to 1.15.0

Fred-sun commented on galaxy.yml:
```diff
@@ -9,7 +9,7 @@ namespace: azure
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.14.0
+version: 1.15.0
```
Please keep this version, we will only change this version when the version is released! Thanks!
```suggestion
version: 1.14.0
```

kmj251 committed:
Update galaxy.yml

Putting this back. Updated for local testing and forgot to put it back to 1.14.0

kmj251 commented:
@Fred-sun Sorry about that. I changed it for local testing and forgot to put it back. Thanks for catching that!
